Failure On Install

I'm tryng to install pynvml from source and getting a strange error:

```
(cudf-dev) bzaitlen@dgx15:~/GitRepos/pynvml$ pip install -e .
Obtaining file:///home/nfs/bzaitlen/GitRepos/pynvml
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/nfs/bzaitlen/GitRepos/pynvml/setup.py", line 20, in <module>
        version=versioneer.get_version(),
      File "/home/nfs/bzaitlen/GitRepos/pynvml/versioneer.py", line 1480, in get_version
        return get_versions()["version"]
      File "/home/nfs/bzaitlen/GitRepos/pynvml/versioneer.py", line 1420, in get_versions
        assert cfg.tag_prefix is not None, "please set versioneer.tag_prefix"
    AssertionError: please set versioneer.tag_prefix
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /home/nfs/bzaitlen/GitRepos/pynvml/
```

PR #13 adds versioneer but maybe something wonky happened in the setup ?

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,2 +1,2 @@
 include versioneer.py
-include pynvml/*.py
+include pynvml/_version.py

--- a/setup.cfg
+++ b/setup.cfg
@@ -8,4 +8,6 @@ VCS = git
 style = pep440
 versionfile_source = pynvml/_version.py
 versionfile_build = pynvml/_version.py
+tag_prefix =
+parentdir_prefix = pynvml-
 

